metadata.json

```
@@ -10,11 +10,11 @@
   "dependencies": [
     {
       "name": "puppetlabs/stdlib",
-      "version_requirement": ">= 4.6.0 < 8.0.0"
+      "version_requirement": ">= 4.6.0 < 10.0.0"
     },
     {
-      "name": "herculesteam/augeasproviders_sysctl",
-      "version_requirement": ">= 2.2.1 < 3.0.0"
+      "name": "hpuppet-augeasproviders_sysctl",
```

```suggestion
      "name": "puppet-augeasproviders_sysctl",
```

I think the `h` is an copy&paste error, can you confirm?

---

Ah, yeah.  I'll fix that.